assets/js/googlesitekit/datastore/user/permissions.js

```diff
@@ -231,6 +231,36 @@ export const selectors = {
 			return undefined;
 		}
 	),
+
+	/**
+	 * Checks if the specified module is shareable and viewable by the current user.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state      Data store's state.
+	 * @param {string} moduleSlug Module slug to check.
+	 * @return {(boolean|undefined)} `true` if the module is shareable and viewable by the current user.
+	 * 								 `false` if the module does not exist, is not shareable or not viewable by the current user.
+	 * 								 `undefined` if state is not loaded yet.
```

Comment:
The indentation here seems a bit off here and in my editor:

<img width="762" alt="CleanShot 2022-06-02 at 20 18 59" src="https://user-images.githubusercontent.com/90871/171720428-d7ec73eb-bf96-419f-a78b-3646d796240f.png">

Sorry the linter isn't catching it, but can you tweak it? 😅